### ISSUE TITLE
OperatorPrecedence too strict for simple expressions

### ISSUE THREAD

**Issue by bobtiernay-okta** [google/error-prone#915]
Please answer these questions before submitting your issue. Thanks!

### What version of Error Prone are you using?

2.2.0

### Does this issue reproduce with the latest release?

Yes

### What did you do?

```
MyFile.java:80: warning: [OperatorPrecedence] Use grouping parenthesis to make the operator precedence explicit
        return interval.isAfter(startMillis) && interval.isBefore(endMillis)
                                             ^
    (see http://errorprone.info/bugpattern/OperatorPrecedence)
  Did you mean 'return (interval.isAfter(startMillis) && interval.isBefore(endMillis))'?
```

### What did you expect to see?

It seems unnecessary to have this advice for this instance

### What did you see instead?

It skip this observation

**Comment by cushon** [comment]
Can you share the rest of the file, or the full line that the diagnostic was emitted for?

**Comment by bobtiernay-okta** [comment]
Oh, wow. I stand corrected:

```java
    public boolean isBetween(long startMillis, long endMillis) {
        return interval.isAfter(startMillis) && interval.isBefore(endMillis)
                || interval.contains(startMillis) || interval.contains(endMillis);
    }
```

Sorry to bother!

Will close.
